interface/src/avatar/MyCharacterController.cpp

```diff
@@ -407,13 +407,13 @@ void MyCharacterController::clearDetailedMotionStates() {
 }
 
 void MyCharacterController::buildPhysicsTransaction(PhysicsEngine::Transaction& transaction) {
-    for (size_t i = 0; i < _detailedMotionStates.size(); i++) {
```

Comment:
This doesn't appear to make any functional change, and to just be a cleanup.

Perhaps this should be in a separate PR?

Reply:
these are leftover from when I was trying to track down the problem.  since this is a short PR and these changes theoretically improve performance, I would vote to keep them